woocommerce/woocommerce-android
# Bump stripe terminal sdk version

Fixes #4288

This PR updates Stripe Terminal SDK version from 1.0.18 to 1.0.22 (version 1.0.19-1.0.21 do not exist)

[The official release notes](https://github.com/stripe/stripe-terminal-android/blob/master/CHANGELOG.md) for 1.0.22 mention that the only change is an update of vendor SDK (I assume card reader vendor). The new version of the vendor SDK fixes some issues - eg. onUnexpectedDisconnect wasn't being emitted when the user turned off bluetooth while connected to a reader -> the SDK remained in "connected" state.


To test:
1. Open app settings
2. Tap on "Manage Card Reader"
3. Connect to a hw reader
4. Wait until connected state is shown
5. Turn off bluetooth and notice you get automatically redirected to the "not connected" screen


Update release notes:
